substrate/frame/revive/rpc/src/client.rs

```diff
@@ -165,7 +164,7 @@ impl From<ClientError> for ErrorObjectOwned {
 
 /// The number of recent blocks maintained by the cache.
 /// For each block in the cache, we also store the EVM transaction receipts.
-pub const CACHE_SIZE: usize = 10;
+pub const CACHE_SIZE: usize = 256;
```

Comment:
How did you come up with this number? Might be worth to explain it in above comment.

Reply:
just a random number :)
that's how many block we keep in the cache for answering eth json-rpc requests, 10 was probably too low
so bumping it just to be safe, I will probably revisit that later

Reply:
What is the memory used in the worst-case scenario for caching one block?

Reply:
did not check but it should be pretty light, we can adapt this code later on so that it does not grow more than let's say 100mb